.github/workflows/integration-tests.yml

```
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v5
       - uses: actions/setup-go@v5
         with:
-          go-version: "1.24"
+          go-version: "1.25"
       - name: Build
         run: go test -run "TestIntegration" -v ./integration_test.go
-        working-directory: "tests/"
+        working-directory: "tests/"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)